site-permissions/site-permissions-impl/src/main/res/layout/dialog_site_permissions.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Copyright (c) 2022 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:background="?toolbarBgColor"
+    android:orientation="vertical">
+
+    <FrameLayout
+        android:id="@+id/sitePermissionsDialogFaviconContainer"
+        android:layout_width="40dp"
+        android:layout_height="40dp"
+        android:layout_gravity="center"
+        android:layout_marginTop="24dp"
+        android:background="@drawable/list_item_image_background">
+
+        <ImageView
+            android:id="@+id/sitePermissionDialogFavicon"
+            android:layout_width="22dp"
+            android:layout_height="22dp"
+            android:layout_gravity="center"
+            android:importantForAccessibility="no"
+            android:src="@drawable/ic_globe_gray_16dp" />
+    </FrameLayout>
+
+    <TextView
+        android:id="@+id/sitePermissionsDialogTitle"
+        style="@style/LocationPermissionDialogTitle"
```

Comment:
I used the Location ones because it is "the same feature". I think it's changed in following stacked PRs where I called it SitePermissionDialogTitle. However, this dialog layout was removed yesterday in the last stacked PR as a part of the design review 😅